.github/workflows/spelling.yml

```diff
@@ -56,37 +80,72 @@ jobs:
       contents: read
       pull-requests: read
       actions: read
+      security-events: write
```

Comment:
We're ok to keep this one - can you just confirm what type of security events are going to be pushed, so I know what to expect? Is it one per misspelling?

Reply:
It's one per instance, they shouldn't be flagged as _security_.
<img width="834" alt="image" src="https://github.com/nasa/fprime/assets/2119212/b242350b-b726-4b4e-a853-0dce682eb1ed">

Note that if you have two instances of the same word in a line you'll get to events -- the web ui and anything that interacts with the sarif reports will show you precisely which substring each event is tagging.